### ISSUE TITLE
"Click here" links

### ISSUE THREAD

**Issue by Steve-Noble** [benetech/AccessibleImageSampleBook#6]
In the EPUB version we repeatedly use the phrase "To zoom, double-tap the image or click here" -- and the text "click here" is used as the descriptive link. That's not a good accessibility practice, and I suggest that we don't model the practice in the Sample Book. [See [What’s wrong with using "Click Here" links?](https://webaccess.berkeley.edu/ask-pecan/click-here) for background.]

**Comment by clapierre** [comment]
You are absolutely right Steve, this book needs a lot of love, and was done before my time.  With DIAGRAM ending not sure if we have the budget for a massive change, we have a few other issues especially with the web version of this as there are some presentational issues with overlapping text :(
This will definitely be schedule as a must fix the next time we make improvements to this. 
Thanks for flagging.
